dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -732,6 +732,13 @@ def test_reductions(split_every):
             bias_factor = (n * (n - 1)) ** 0.5 / (n - 2)
             assert_eq(dds.skew(), pds.skew() / bias_factor)
 
+        if scipy:
+            # pandas uses a bias factor for kurtosis, need to correct for that
```

Comment:
Should dask also have a bias factor to mimic pandas?

Reply:
The spot to implement that would be [here](https://github.com/dask/dask/blob/master/dask/array/stats.py#L259-L261). Judging from the comment this wasn't easily doable when the kurtosis method was added to dask.array.